src/perfetto_cmd/perfetto_cmd.h

```diff
@@ -109,21 +109,28 @@ class PerfettoCmd : public Consumer {
 
   void ReadbackTraceDataAndQuit(const std::string& error);
 
-  enum BgProcessStatus : char {
-    kBackgroundOk = 0,
-    kBackgroundOtherError = 1,
-    kBackgroundTimeout = 2,
+  enum WaitStatus : char {
+    kWaitOk = 0,
+    kWaitOtherError = 1,
+    kWaitTimeout = 2,
   };
 
-  void Notify(BgProcessStatus status);
+  // Used to implement the --notify-fd flag.
+  //
+  // Signals client by writing to FD (if there is one) that data sources has
+  // started (or failed to start).
+  //
+  // Only the first time this function is called is significant. Further calls
+  // will have no effect.
+  void NotifyFd(WaitStatus status);
```

Comment:
Thank you :) I wanted to suggest this (+"Fd") in the other CL, but then I felt we went too much in circles already and was not big enough .